open_src/github.com/VictoriaMetrics/VictoriaMetrics/lib/fs/fadvise_windows.go

```diff
@@ -0,0 +1,22 @@
+//go:build windows
+// +build windows
+
+package fs
+
+import (
+	"fmt"
+
+	"github.com/openGemini/openGemini/lib/fileops"
+)
+
+func fadviseSequentialRead(f fileops.File, prefetch bool) error {
+	fd := int(f.Fd())
```

Comment:
It does not need to be implemented. Directly return nil.

Reply:
Yes, I agree. The `fadviseSequentialRead` not really used at all.